packages/components/atoms/f-image-tile/stories/Demo.stories.js

```diff
@@ -32,12 +36,17 @@ DemoImageTileComponent.argTypes = {
     cuisineImage: {
         control: { type: 'text' },
         description: 'Cuisine image link',
-        defaultValue: 'https://via.placeholder.com/150'
+        defaultValue: ''
     },
     cuisineId: {
         control: { type: 'text' },
         description: 'Cuisine id',
         defaultValue: 'Chicken'
+    },
+    fallbackImage: {
+        control: { type: 'text' },
+        description: 'Fallback image link',
+        defaultValue: 'https://d30v2pzvrfyzpo.cloudfront.net/a/sw/img/wallpaper.png'
```

Comment:
something useful that was recommended to me for f-restaurant-card is to self-host an image and import into the file, rather than rely on an external source.

In case that sounds like something you'd be interested in I'll link the implementation [here](https://github.com/justeat/fozzie-components/blob/41c6f5f62d1b6b4e69daafd35f61bcd2caf69951/packages/components/molecules/f-restaurant-card/stories/RestaurantCard.stories.js#L8) and [here](https://github.com/justeat/fozzie-components/blob/41c6f5f62d1b6b4e69daafd35f61bcd2caf69951/packages/components/molecules/f-restaurant-card/stories/RestaurantCard.stories.js#L31) :)

Reply:
Ahh perfect, will take a look and implement thank you!

Reply:
Awesome, I've updated now 🎇